.github/actions/ci/action.yml

```diff
@@ -35,4 +43,4 @@ runs:
       shell: bash
       run: |
         dotnet restore ${{ inputs.test_project_file }}
-        dotnet test --framework=net6.0 ${{ inputs.test_project_file }}
+        dotnet test -v=${{ inputs.test_verbosity }} --framework=${{ inputs.target_test_framework }} ${{ inputs.test_project_file }}
```

Comment:
Previously, it was passing net6 as the framework. I believe this was causing it to silently just not run tests at all. The target for the client tests is net7.